modules/users/manifests/davidgisbey.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the davidgisbey user
+class users::davidgisbey {
+  govuk_user { 'davidgisbey':
+    fullname => 'David Gisbey',
+    email    => 'david.gisbey@digital.cabinet-office.gov.uk',
+    ssh_key  => 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAKjjPnumAMyuz3E3vKGavYaPEoHM9e9mjPyfXqD9UNX davidgisbey@gmail.com',
```

Comment:
You might want to use a work email here, as this is a public repo

Reply:
Thanks. Have updated